recipes/flang/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set clang_variant = os.environ.get('CLANG_VARIANT', 'flang') %}
+{% set flang_commit = "ab1423f97274b472970efd53e01800a6d4f137f5" %}
+{% set version = "5.0.0" %}
+{% set sha256 = "01264da25bac23e27ca9298bbfd41f7226854247e92a3edea6f28decf42c7b09" %}
+{% set build_number = "0" %}
+
+package:
+  name: flang
+  version: {{ version }}
+
+source:
+  url: https://github.com/isuruf/flang/archive/{{ flang_commit }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: {{ build_number }}
+  skip: true    # [osx]
+  skip: true    # [win32]
```

Comment:
You need to skip Python 2.7. There's no point in adding support because the ABI is incompatible.

Reply:
Also, why is win32 skipped?

Reply:
Flang only supports 64-bit right?

Reply:
There's no python in the build requirements. just `msvc_compiler: 14.0`.

Reply:
In theory yes (although I would like to understand why this is in the future). However, to me "win32" means both 32 and 64 bit windows. Maybe it's different for conda.

Reply:
So the test failures are okay then?